### PR TITLE
Extend on Prometheus file SD

### PR DESCRIPTION
**Pull request by witekest**
# Description

Add more details about Prometheus file service discovery.

# Target branches

Which documentation version does this PR apply to?

- [x] Master (Default)
- [x] #1019 
- [x] #1020

# Links

https://github.com/SUSE/spacewalk/issues/15157

### CODE REVIEW

**Review by jcayouette — APPROVED** [review]

File: modules/salt/pages/formula-monitoring.adoc [hunk 1]
```diff
@@ -68,9 +68,9 @@ For more information about using monitoring, see xref:administration:monitoring.
 . To use a rule file, in the menu:Alerting[Rule Files] section, click btn:[+], and set these parameters:
 * In the [guimenu]``Rule Files`` field, type the location of the rule file you want to use.
 //For example, ``FIXME``.
-. To use a scrape configuration, in the [guimenu]``User defined scrape configurations`` section, click btn:[+], and set these parameters:
-* In the [guimenu]``Job name`` field, type a name for your configuration.
-* In the [guimenu]``Files`` field, type the location of the configuration file you want to use.
+. To add a custom scrape configuration, in the [guimenu]``User defined scrape configurations`` section, click btn:[+], and set these parameters:
+* In the [guimenu]``Job name`` field, type a unique job name for your configuration.
+* In the [guimenu]``Files`` field, type the location pattern of file service discovery files you want to use. For more information refer to upstream documentation https://prometheus.io/docs/prometheus/latest/configuration/configuration/#file_sd_config.
```
Comment (keichwa) [inline modules/salt/pages/formula-monitoring.adoc:73]:
```suggestion
* In the [guimenu]``Files`` field, type the location pattern of file service discovery files you want to use.
  For more information, see the upstream documentation https://prometheus.io/docs/prometheus/latest/configuration/configuration/#file_sd_config.
```
minor normalization according to our style and formatting rules.

**Review by keichwa — APPROVED** [review]

**Comment by witekest** [comment]
Could you please merge if appropriate. I don't have powers.

**Comment by keichwa** [comment]
> Could you please merge if appropriate. I don't have powers.

Do we also want to have it backported to 4.2?

**Comment by keichwa** [comment]
thanks, merge done.

**Pull request merged**